client/my-sites/invites/invite-accept-logged-in/index.jsx

```diff
@@ -21,6 +22,14 @@ export default React.createClass( {
 
 	displayName: 'InviteAcceptLoggedIn',
 
+	acceptInvite() {
+		const { invite } = this.props;
+		acceptInvite( invite, () => {
+			page( this.props.redirectTo );
```

Comment:
It doesn't look like we're doing any error handling here. I understand it's probably more edge case to handle errors here (since we validate an invite before showing the user this UI), but it is possible that a user is mobile and lost connection or somehow opened another tab and took an action which invalidated an invite.

In this case, we shouldn't redirect the user and tell him/her that it was successful.